### ISSUE TITLE
Thanks for your Great work!

### ISSUE THREAD

**Issue by Isi-dev** [antgroup/animate-x#1]
Thanks for your great work. I have done some tests with animate-x by making a little modification to my code for ComfyUI-Unianimate-w and integrating the animate-x_ckpt.pth model. Most of the results were great. But, there were a few issues like parts of the female hair  sometimes being ignored in the animation, and multiple hands sometimes appearing in non-human characters. Can you give any advice on this?
I will also like to know the implication of `del state_dict[key]` & `status = model.load_state_dict(state_dict, strict=False)` in the  following section (lines 366 - 373) of the code  in animatex/inference_animate_x_entrance.py:

```
    try:
        status = model.load_state_dict(state_dict, strict=False)
    except:
        for key in list(state_dict.keys()):
            if 'pose_embedding_before.pos_embed.pos_table' in key:  
                del state_dict[key]
        status = model.load_state_dict(state_dict, strict=False)
```

Thanks.

**Comment by tanshuai0219** [comment]
Thanks for your interest in Animate-X
For the first issue: You can try using different random seeds; you can specify this in https://github.com/antgroup/animate-x/blob/7f74026ca183151735e47da507bacf3d8ba7ec4c/configs/Animate_X_infer.yaml#L10

The last parameter indicates the random seed.

For the second issue: The code you mentioned will only take effect when the number of generated frames is not equal to 32 frames.

**Comment by Isi-dev** [comment]
> Thanks for your interest in Animate-X For the first issue: You can try using different random seeds; you can specify this in
> 
> https://github.com/antgroup/animate-x/blob/7f74026ca183151735e47da507bacf3d8ba7ec4c/configs/Animate_X_infer.yaml#L10
> 
> The last parameter indicates the random seed.
> 
> For the second issue: The code you mentioned will only take effect when the number of generated frames is not equal to 32 frames.

Okay. Thank you.

**Comment by tanshuai0219** [comment]
> Thanks for your great work. I have done some tests with animate-x by making a little modification to my code for ComfyUI-Unianimate-w and integrating the animate-x_ckpt.pth model. Most of the results were great. But, there were a few issues like parts of the female hair sometimes being ignored in the animation, and multiple hands sometimes appearing in non-human characters. Can you give any advice on this? I will also like to know the implication of `del state_dict[key]` & `status = model.load_state_dict(state_dict, strict=False)` in the following section (lines 366 - 373) of the code in animatex/inference_animate_x_entrance.py:
> 
> ```
>     try:
>         status = model.load_state_dict(state_dict, strict=False)
>     except:
>         for key in list(state_dict.keys()):
>             if 'pose_embedding_before.pos_embed.pos_table' in key:  
>                 del state_dict[key]
>         status = model.load_state_dict(state_dict, strict=False)
> ```
> 
> Thanks.

Additionally, as we mentioned in `Tips`：
https://github.com/antgroup/animate-x/blob/fdc80909f911d8e487cb4e6847f2e4c6501b62af/README.md?plain=1#L156
Providing Animate-X with the aligned pose images with reference image will achieve a better performance.

For the alignment script, you can refer to [pose_align.py](https://github.com/TMElyralab/MusePose/blob/main/pose_align.py).

**Comment by Isi-dev** [comment]
> > Thanks for your great work. I have done some tests with animate-x by making a little modification to my code for ComfyUI-Unianimate-w and integrating the animate-x_ckpt.pth model. Most of the results were great. But, there were a few issues like parts of the female hair sometimes being ignored in the animation, and multiple hands sometimes appearing in non-human characters. Can you give any advice on this? I will also like to know the implication of `del state_dict[key]` & `status = model.load_state_dict(state_dict, strict=False)` in the following section (lines 366 - 373) of the code in animatex/inference_animate_x_entrance.py:
> > ```
> >     try:
> >         status = model.load_state_dict(state_dict, strict=False)
> >     except:
> >         for key in list(state_dict.keys()):
> >             if 'pose_embedding_before.pos_embed.pos_table' in key:  
> >                 del state_dict[key]
> >         status = model.load_state_dict(state_dict, strict=False)
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Thanks.
> 
> Additionally, as we mentioned in `Tips`：
> 
> https://github.com/antgroup/animate-x/blob/fdc80909f911d8e487cb4e6847f2e4c6501b62af/README.md?plain=1#L156
> 
> 
> Providing Animate-X with the aligned pose images with reference image will achieve a better performance.
> For the alignment script, you can refer to [pose_align.py](https://github.com/TMElyralab/MusePose/blob/main/pose_align.py).

Okay, thanks.
